content/building/build-versioning.md

```diff
@@ -46,3 +46,112 @@ set -x
 cd $FCI_BUILD_DIR/ios
 agvtool new-version -all $(($BUILD_NUMBER + 1))
 ```
+
+## Use Codemagic CLI Tools to get the latest build number from App Store or TestFlight
+
+Use [get-latest-app-store-build-number](https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/app-store-connect/get-latest-app-store-build-number.md#get-latest-app-store-build-number) or [get-latest-testflight-build-number](https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/app-store-connect/get-latest-testflight-build-number.md#get-latest-testflight-build-number) actions from [app-store-connect](https://github.com/codemagic-ci-cd/cli-tools/tree/master/docs/app-store-connect#app-store-connect) Codemagic CLI Tool to get the latest build numbers.
+
+In order to do that, you need to provide API access to App Store Connect API by providing `ISSUER_ID`, `KEY_IDENTIFIER` and `PRIVATE_KEY` as arguments to the action, similarly how it's done while [Signing iOS apps](https://docs.codemagic.io/code-signing-yaml/signing-ios/).
```

Comment:
I would remove `action, similarly how it’s done while Signing iOS apps` from this paragraph and replace `action, as defined below`. I got more confused trying to go to this link to see how it would help me. You already copy and pasted the important things of codesigning here.

Reply:
fixed [here](https://github.com/codemagic-ci-cd/codemagic-docs/pull/267/commits/df25906cd72dda60752b995da4188ab9787c3146#diff-f6e40a5795c4d7de47d17ed5b2850ac8a06b55f09f22d3bf4611e3a4bfbffe4bR54) and [here](https://github.com/codemagic-ci-cd/codemagic-docs/pull/267/commits/df25906cd72dda60752b995da4188ab9787c3146#diff-f6e40a5795c4d7de47d17ed5b2850ac8a06b55f09f22d3bf4611e3a4bfbffe4bR135)